code/script.js

```diff
@@ -30,18 +55,134 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+//Interaction 7, confirm all preferenses and suggest dog breeds.
+const dogBreed = (lifestyle, size, live, coat) => {
+  if(lifestyle === "couchPotato" && size === "small" && live === "citySuburb" && coat === "short" ) {
+  showMessage("Based on your preferences;<br>You're a Couch potato who live in a city or suburb and wants a small sized dog with a short coat.<br> You should get a short coated Chihuahua or a Pug!", "bot")
+  } else if(lifestyle === "couchPotato" && size === "small" && live === "citySuburb" && coat === "long" ) {
+    showMessage("Based on your preferences;<br>You're a Couch potato who live in a city or suburb and wants a small sized dog with a long coat.<br> You should get a long coated Chihuahua or a Pekingese!", "bot")
+  } else if(lifestyle === "couchPotato" && size === "small" && live === "countryside" && coat === "short" ) {
+    showMessage("Based on your preferences;<br>You're a Couch potato who live in a city or suburb and wants a small sized dog with a long coat.<br> You should get a short coated Chihuahua or a Pekingese!", "bot")
+  } else {
+  showMessage("I need to think about this a bit, come back later!", "bot")
+  }
+}
+
+//Interaction 6, confirm environment choice and ask what coat they prefer on a dog. Store value.
+const coatChoice = (live) => {
+  showMessage(`Ok, ${live} it is! Do you want a dog with a short or long coat?`, "bot")
+    inputWrapper.innerHTML = buttonSelectCoat
+    document
+      .getElementById("long")
+      .addEventListener("click", () => {
+      showMessage("I want a dog with a long coat", "user")
+      inputWrapper.innerHTML = ""
+      setTimeout(() => dogBreed("long"), 1000)
+  })
+  document
+      .getElementById("short")
+      .addEventListener("click", () => {
+      showMessage("I want a dog with a short coat", "user")
+      inputWrapper.innerHTML = ""
+      setTimeout(() => dogBreed("short"), 1000)
+  })
+  const coat = document.getElementById("submit")
+}
+
+//Interaction 5, confirm size choice and ask if they live in city/suburb or countryside. Store value.
+const environmentChoice = (size) => {
+  showMessage(`Ok, you you want a ${size} sized dog breed. Where do you live?`, "bot")
+  inputWrapper.innerHTML = buttonSelectEnvironment
+  document
+    .getElementById("citySuburb")
+    .addEventListener("click", () => {
+    showMessage("I live in a city or suburb", "user")
+    inputWrapper.innerHTML = ""
+    setTimeout(() => coatChoice("citySuburb"), 1000)
+  })
+    document
+    .getElementById("countryside")
+    .addEventListener("click", () => {
+    showMessage("I live in the countryside", "user")
+    inputWrapper.innerHTML = ""
+    setTimeout(() => coatChoice("countryside"), 1000)
+  })
+  const live = document.getElementById("submit")
+}
```

Comment:
**5.** Then you would need to pick up both arguments here:
```const environmentChoice = (lifestyleChoice, size) => {```
and when you call on the next function you need to pass along three arguments:
```setTimeout(() => coatChoice(lifestyleChoice, size, "countryside"), 1000)```